doc/authoring_tests.md

```diff
@@ -1,27 +1,44 @@
-Automating tests for Azure CLI
-========================================
+# Automating tests for Azure CLI
 
 ## Overview
 
 There are two types of automated tests you can add for Azure CLI: [unit tests](https://en.wikipedia.org/wiki/Unit_testing) and  [integration tests](https://en.wikipedia.org/wiki/Integration_testing).
 
-For unit tests, we support unit tests written in the forms of both standard [unittest](https://docs.python.org/3/library/unittest.html) and [nosetest](http://nose.readthedocs.io/en/latest/writing_tests.html).
+For unit tests, we support unit tests written in the forms standard [unittest](https://docs.python.org/3/library/unittest.html). The current support of [nosetest](http://nose.readthedocs.io/en/latest/writing_tests.html) will be gradually retired.
```

Comment:
This grammar is wonky.  Do we even have any tests written in "nosetest form"?  If not, we can just erase mention of it.

Reply:
I will remove it then.